test/sql/copy/csv/test_multiple_big_compressed_csvs.test_slow

```diff
@@ -4,9 +4,6 @@
 
 require tpch
 
-statement ok
```

Comment:
Was that accidental?

Reply:
No this was on purpose - the test takes a really long time already